Readme.adoc

```diff
@@ -116,6 +116,38 @@ If you want to use multiple filters at once, it's possible to separate them with
 [...]
 ----
 
+=== Job to check skip message
```

Comment:
Personally I would put this one at the top, above all other methods. It behaves most similarly to how `ci skip` does on all other platforms with fewer drawbacks.

Reply:
Thank you for your comment. Let's merge as it is for now. The example is a little bit long what can be a problem for first-time users.